specs/DateTimePicker-Visual-Updates-Spec.md

```diff
@@ -0,0 +1,228 @@
+is an element rather than a Brush. So it can't for example be rendered as text foreground.
+
+This new `MonochromaticOverlayPresenter` will be used in the rendering of the 
+[DatePicker](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.DatePicker)
+control.
+
+# API Pages
+
+## MonochromaticOverlayPresenter class
+
+A [Framework Element](https://docs.microsoft.com/windows/winui/api/microsoft.ui.xaml.frameworkelement?view=winui-3.0-preview)
+whose background can be the rendering of another element, or using another element
+as a mask.
+
+```csharp
+public class MonochromaticOverlayPresenter : FrameworkElement
+{
+    public UIElement SourceElement { get; set; }
+    public Color ReplacementColor { get; set; }
+}
+```
+
+Set the `SourceElement` property to indicate which element's rendering should be drawn as the background of the `MonochromaticOverlayPresenter`. If the `Background` property is set, the SourceElement's rendering is on top of it. (The source is z-ordered on top of the background.
+
+Only the source element's rendering applies to the `MonochromaticOverlayPresenter`; the user
+cannot interact with it using the mouse, or move keyboard focus to it. However, `MonochromaticOverlayPresenter` will bring in system focus visuals, regardless of what the [UseSystemFocusVisuals](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.Control.UseSystemFocusVisuals) property is set to. To avoid this, you should inset the margins to remove it.
+
+> Note: the element set as the `SourceElement` cannot be an ancestor or in the descendancy of the `MonochromaticOverlayPresenter`. However the `SourceElement` must be in the same XAML tree as the `MonochromaticOverlayPresenter`.
+
+
+If the `ReplacementColor` property is set, the `SourceElement` is treated as a mask; every non-transparent pixel will be replaced by this color. The source pixel is replaced when the Alpha value of the `ReplacementColor` is non-zero. The RGB value of the `ReplacementColor` (not the Alpha value) will overwrite the RGB of the source. The default value for `ReplacementColor` is Transparent.
+
+
+### MonochromaticOverlayPresenter examples
+
+The following example puts an `MonochromaticOverlayPresenter` on top of (z-ordered above)
+some scrolling text, causing the center of the text to be highlighted, even as it's scrolling.
+Things to note:
+* The `MonochromaticOverlayPresenter` is sourcing from the `TextBlock`, it's also z-ordered above
+the same `TextBlock`. If the `Background` and `SourceElement` properties weren't set,
+it would have no effect; it would render the `TextBlock` exactly on top of itself.
+* The 'Background` property is black, and the `ReplacementColor` is green, causing
+the overlay text to be displayed as green on black.
+* The `MonochromaticOverlayPresenter` is positioned and given a height such that it's a black band
+across the center of the scrolling text.
```

Comment:
Its `IsHitTestVisible` property is set to `False` so that input still goes to the `ScrollViewer` that is z-ordered behind it.